exercises/alphametics/example.rb

```diff
@@ -0,0 +1,41 @@
+class Alphametics
+  VERSION = 1
+
+  def solve(puzzle)
+    letters = Hash.new(0)
+    puzzle = format_to_ruby(puzzle)
```

Comment:
I don't think the eval is a good idea, so yeah< I think that is good.

I just noted that there wasn't anything stating that the formulas needed to be directly evaluate-able.